modules/distribution/src/repository/jaggeryapps/cdm/modules/sso.js

```diff
@@ -71,12 +77,15 @@ var ssoMod = require("sso");
  		sso.relayState = (referer ? referer : sso.relayState);
  		sso.relayState = sso.relayState + request.getQueryString(); // append query string
  		sso.encodedSAMLLogoutRequest = ssoMod.client.getEncodedSAMLLogoutRequest(user, sso.sessionIndex , sso.issuer);
- 		sso.log.debug("Logout request recieved from session id ###: " + sso.sessionId );
- 		
  		var postUrl = sso.address + sso.ssoService;
- 		sso.log.debug(sso.sessionId);
+
+		if(log.isDebugEnabled()){
+			sso.log.debug("Logout request recieved from session id ###: " + sso.sessionId );
+			sso.log.debug(sso.sessionId);
```

Comment:
Session id seems to have been logged already in the previous line. Do we still need to have another separate line for the same?